csaf/src/rolie/roliefeed.rs

```diff
@@ -0,0 +1,71 @@
+use serde::{Deserialize, Serialize};
+use time::OffsetDateTime;
+
+#[derive(Serialize, Deserialize)]
+pub struct RolieFeed {
+    pub feed: Feed,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct Feed {
+    pub entry: Vec<Entry>,
```

Comment:
For this and other field of type `Vec<_>`, are they required? Or can they be empty. If they can, it requires the `#[serde(default)]` annotation as well as `#[serde(skip_serializing_if="Vec::is_empty")]`.